### ISSUE TITLE
Add tags and tag categories

### ISSUE THREAD

**Issue by dbrandonjohnson** [lakoliu/Furtherance#8]
Being able to and report on tasks with tags makes it easier for filtering and reporting.  Having tags separated by category would be nice as well if wanting to use the same tag with different clients or separate work tasks from personal tasks that may have the same tag or even the same name for the tasks.

**Comment by lakoliu** [comment]
@dbrandonjohnson thanks for the suggestion! Tags are something I already have in the pipeline, but I haven't yet figured out the best way to implement them in terms of sorting tasks by tag, etc. However, I am working on this and hope it will be released within the next couple of months. Any specific ideas on how you would like them displayed would be appreciated!

**Comment by lakoliu** [comment]
Tags have been added to Furtherance and are in the 1.1.2 release! As of now, they don't actually do anything, but my next big improvement to Furtherance will be "Reports" (Issue #32) and that is when tags will become useful, as you'll be able to sort by tags and create reports based on tags. I also did not implement categories, as I didn't see a great way to do so, but perhaps in the future they will be introduced.
